<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtclickerClass</class>
 <widget class="QMainWindow" name="qtclickerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qtclicker</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 141, 5);</string>
   </property>
   <widget class="QFrame" name="frameButtons">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>230</y>
      <width>230</width>
      <height>111</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="buttonGo1">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(60, 182, 4);
color: rgb(122, 84, 53);</string>
       </property>
       <property name="text">
        <string>Go 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonGo2">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(60, 182, 4);
color: rgb(122, 84, 53);</string>
       </property>
       <property name="text">
        <string>Go 2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonExit">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(60, 182, 4);
color: rgb(122, 84, 53);</string>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frameScore">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>0</y>
      <width>230</width>
      <height>70</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <layout class="QVBoxLayout" name="layoutScore">
     <item>
      <widget class="QProgressBar" name="progressBarTime">
       <property name="value">
        <number>24</number>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelScore">
       <property name="text">
        <string>10</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frameReturn">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>160</y>
      <width>230</width>
      <height>70</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="labelLastScore">
       <property name="text">
        <string>Your score: 20 / 30</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonReturn">
       <property name="text">
        <string>Return</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="frameDots">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>79</y>
      <width>550</width>
      <height>450</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(170, 85, 0);
border-width: 5px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
   </widget>
   <zorder>frameDots</zorder>
   <zorder>frameButtons</zorder>
   <zorder>frameScore</zorder>
   <zorder>frameReturn</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuhelp">
    <property name="title">
     <string>help</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuoptions">
    <property name="title">
     <string>options</string>
    </property>
   </widget>
   <addaction name="menuoptions"/>
   <addaction name="menuhelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
